backend/infrahub/events/tasks.py

```diff
@@ -0,0 +1,21 @@
+from prefect import flow
```

Comment:
I would suggest to move this file, either under `git/tasks.py` or to create a new directory named `artifacts`
I was planning to reserve event the definition of the events themselves and to have them separated from the workflows

Reply:
Thanks, I was not sure where to locate it and put it here by waiting to have more information about it